src/AimInputComponent.h

```diff
@@ -0,0 +1,24 @@
+#ifndef AIM_INPUT_COMPONENT_H
+#define AIM_INPUT_COMPONENT_H
+
+#include "InputComponent.h"
+
+class AimInputComponent : public InputComponent {
```

Comment:
I literally thought "this seems perfect for the input component" and then it already was, awesome lol